schemas/amulet/20210221.json

```diff
@@ -0,0 +1,69 @@
+{
+    "description": "This schema describes version 20210221 of the metadata standard for the amulet poem format, designed for the Zora protocol.",
+    "title": "Amulet20210221",
+    "$id": "http://text.bargains/schema.json",
+    "type": "object",
+    "properties": {
+        "description": {
+            "description": "The description of the amulet, which should include at minimum (1) a link to the current formal definition and (2) a link to the carbon offset purchased in the poem's name.",
+            "examples": [
+                "This is an amulet, a short poem with a lucky SHA-256 hash, explained here: https://text.bargains/ This poem's rarity is <rarity>. Here is this poem's carbon offset: <link>"
+            ],
+            "type": "string"
+        },
+        "mimeType": {
+            "description": "The mimeType of the amulet. This will always be text/plain.",
```

Comment:
if its always going to be text/plain you should make this an enum! Check catalog's schema for inspo https://github.com/ourzora/media-metadata-schemas/blob/master/schemas/catalog/20210202.json#L91

Reply:
Will do! Thank you.